web/package.json

```diff
@@ -21,6 +21,9 @@
         "precommit": "run-s tsc lit-analyse lint:precommit lint:spelling prettier",
         "prettier-check": "prettier --check .",
         "prettier": "prettier --write .",
+        "pseudolocalize:build-extract-script": "cd scripts && tsc --esModuleInterop --module es2020 --moduleResolution 'node' pseudolocalize.ts && mv pseudolocalize.js pseudolocalize.mjs",
```

Comment:
Running scripts in this format requires ES Module format, which Node requires be suffixed as `.mjs`.

It's a crime that package.json doesn't support comments.